kolibri/locale/supported_languages.json

```diff
@@ -20,6 +20,12 @@
     "language_name": "Español",
     "english_name": "Spanish"
   },
+  {
+    "language_code": "es",
+    "territory_code": "mx",
+    "language_name": "Español (Mexicano)",
```

Comment:
It could be just me, but the wording `Español (Mexicano)` sounds... not right.

I'd expect a reference to a location (Mexico/Méjico), and not a adjective (Mexican/Mexicano). Something akin to the way Google has it in Search settings:

![languages](https://user-images.githubusercontent.com/1457929/44613723-5dbaba00-a818-11e8-9f5b-4801b9ae25a5.png)

Reply:
cool, thanks

Reply:
Note that we currently use the adjective form for the other languages:

* Portuguese (Brazilian)
* Urdu (Pakistani)
* Farsi (Persian)

A few open questions:

1. I can easily change Portuguese and Urdu .. not sure what to do about Farsi ("Persia"?)
2. Do I change all these now, as part of this PR? Or leave 'mexicano' for now for consistency?
3. Are the language names in their native forms correct? (this is more important than the english name because the english isn't actually displayed to users.) I noticed there are some inconsistencies - for example for Hindi I have english 'Hindi' and native 'हिंदी (भारत)'. I'm guessing this says "Hindi (Indian)" or "Hindi (India)"?

Reply:
cc @carinediaz --^